packages/@coorpacademy-components/src/molecule/skill-card/style.css

```
@@ -57,3 +59,9 @@
 .buttonWrapper {
   padding: 0 24px 24px;
 }
+
+@media mobile {
+  .skillCardWrapper:hover {
+    box-shadow: 0 4px 20px box_shadow_medium_dark;
+  }
+}
```

il manque un saut de ligne (si jamais il y a un pb de lint dans la CI):
```suggestion
}

```